dev/com.ibm.ws.jmx.connector.server.rest/src/com/ibm/ws/jmx/connector/server/rest/handlers/MBeanOperationHandler.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2016 IBM Corporation and others.
+ * Copyright (c) 2016,2023 IBM Corporation and others.
  * All rights reserved. This program and the accompanying materials
```

Comment:
@idlewis wondering whether the repairSlashes method be called within the _deleteMBean_ method? 
https://github.com/OpenLiberty/open-liberty/blob/e9ab3f3e1b5a82af90b47b4730ca3e9b5d7f1280/dev/com.ibm.ws.jmx.connector.server.rest/src/com/ibm/ws/jmx/connector/server/rest/handlers/MBeanHandler.java#L173

Reply:
Yes, I must have missed that one :-( . I'll have to add this and rerun the build.